packages/validation/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@user-office-software/duo-validation",
-  "version": "5.0.2",
+  "version": "5.0.3",
```

Comment:
I think it is better to bump to the new minor version instead of patch. Something like `5.1.0` makes more sense because you are adding some new functionality here. https://semver.org/#:~:text=MAJOR%20version%20when,compatible%20bug%20fixes

Reply:
You will also need to execute `npm run lerna:prePublish` after increasing the version to update the `package-lock.json` file as well.